v2/googlecloud-to-googlecloud/pom.xml

```diff
@@ -62,6 +62,30 @@
     </plugins>
   </build>
 
+  <dependencyManagement>
+    <dependencies>
+      <!-- Enforce Conscrypt version -->
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-bigtable</artifactId>
+        <version>2.25.0</version>
+        <type>jar</type>
+      </dependency>
+      <dependency>
+        <groupId>com.google.api</groupId>
+        <artifactId>gax</artifactId>
+        <version>2.31.1</version>
+        <type>jar</type>
+      </dependency>
```

Comment:
Can you use 2.31.0 please? Based on https://mvnrepository.com/artifact/com.google.cloud/google-cloud-bigtable/2.25.0

And add a TODO to remove this block once libraries-bom gets past `26.19.0`? (likely Beam 2.50.0)